resources/todomvc/vanilla-examples/javascript-es5/index.html

```diff
@@ -1,20 +1,23 @@
-<!doctype html>
-<html lang="en" data-framework="javascript">
+<!DOCTYPE html>
+<html lang="en" data-framework="javascript" data-version="es5" data-features="">
```

Comment:
What are data-features and data-version for?

Reply:
I have to admit, that I saw "data-framework" in the html files and thought that it's an interesting feature to have, although it doesn't look like we're doing anything with it. 
Maybe that could be useful in the future if we want to query during the tests what framework, version and features are currently tested? 

I can totally remove all of those data attributes though